### DISCUSSION TITLE
Is possible to compress a hbitmap?

### DISCUSSION THREAD

**Discussion by pepeu93** [nothings/stb#1280] [Q&A]
I'm not disposing/deleting some hbitmaps as they are used in an animation, this way I reduce the CPU use but increase a lot the use of RAM.

They are loaded into an HDC, is possible somehow 'compress' them to reduce her size in memory?

**Comment by rygorous** [comment]
That's a Windows/GDI+ question that has nothing to do with the stb libs. Try Stack Overflow or similar?

**Reply by pepeu93** [reply]
My question was if the lib support resizing a GDI hbitmap...

**Reply by nothings** [reply]
stb_image etc. work with byte arrays of pixels. if you have byte arrays of pixels, you can use stb to work with them, and we can answer questions about them. but whether an hbitmap is a byte array of pixels, or can be converted to them, is outside the scope of these libraries, which work with portable abstractions like arrays of bytes, not platform-specific things like hbitmaps, and those questions should be asked elsewhere.
